gapic-generator/templates/default/helpers/presenters/method_presenter.rb

```diff
@@ -141,6 +141,32 @@ def routing_params?
     routing_params.any?
   end
 
+  ##
+  # Returns the `GRPC::Core::StatusCodes` that should be retried for this method.
+  #
+  # Strings are returned instead of the actual `GRPC::Core::StatusCodes` values because those constants point to
+  # integers, and then an array of integers would be generated. Returning strings here allows us to generate arrays of
+  # the actual constants which should read better.
+  #
+  # @return [Array<String>] The retry codes
+  #
+  def retry_codes
```

Comment:
Can we move this implementation to the cloud generator? I'll defer to @dazuma if these are sensible defaults in that context (`UNAVAILABLE` likely needs to be removed).

For the base generator, I think it would be better to disable retries and return an empty list here.

Reply:
I have removed UNAVAILABLE.

Why do we want to retry in cloud but not retry in default?

Reply:
Basically because we can't agree on a sensible default retry strategy in the general case yet. For Cloud APIs, we may be able to make some assumptions.  Of course, once the spec is updated we can revisit this.